geometry_msgs/PoseStamped obj_pose
---
bool status